src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,20 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence.
+    //[category: Creating generators]
+    [<CompiledName("Shuffle")>]
+    let shuffle xs =
+        gen {
+            let h = 922337203
```

Comment:
I'm sorry to beat a seemingly dead horse, but as a reader of this code, imagining that I have no prior knowledge of our conversation, I'd still wonder what's so special about this particular number...